src/manifest/dbtProject.ts

```diff
@@ -334,6 +334,9 @@ export class DBTProject implements Disposable {
       await this.python?.lock(
         (python) => python!`to_dict(project.safe_parse_project(${init}))`,
       );
+      // Add dbt packagesInstallPath in dbt modules
+      const packagesInstallPath = await this.findPackagesInstallPath();
+      DBTProject.DBT_MODULES.push(packagesInstallPath);
```

Comment:
If we can reliably retrieve the packagesInstallPath no need for a constant. Please remove the usage of the constant and use `await this.findPackagesInstallPath();`
Also as this involves a lock it might be better to cache all configs in a local variable in the The dbtProject instead. We probably need more configs.